x/group/spec/params.md

```diff
@@ -0,0 +1,8 @@
+# Parameters
+
+The group module contains the following parameters:
+
+| Key                    | Type            | Usage | Example |
+|------------------------|-----------------|---------|
+| MaxVotingPeriod        | string (time ns) | Proposals older than this time will be garbage-collected | "172800000000000"   |
```

Comment:
okay, in general decision policies can't set a voting timeout, smaller than this garbage collection limit.

also, no need for ns precision with ~5s blocktime. Removing 9 zeros, makes this much more legible (it is 2 days, right)

Reply:
Express it in minutes then? I think that precision is for easy conversion to `time.Time`